app/views.py

```diff
@@ -228,7 +228,7 @@ def taskView(request, projectID, taskID):
 
     if "kanban" in request.path:
         baseTemplate = "kanban.html"
-        submitLink = f"/project/{projectID}/kanban/task/{taskID}"
+        submitLink = f"/project/{projectID}/kanban/task/{taskID}/"
         exitLink = f"/project/{projectID}/kanban"
         deleteLink = f"/project/{projectID}/kanban"
     else:
```

Comment:
I would change this so there is defaults set for these variables instead
```suggestion
    baseTemplate = "kanban.html"
    submitLink = f"/project/{projectID}/kanban/task/{taskID}"
    submitLink = f"/project/{projectID}/kanban/task/{taskID}/"
    exitLink = f"/project/{projectID}/kanban"
    deleteLink = f"/project/{projectID}/kanban"
    
    if "kanban" not in request.path:
    	baseTemplate = "graph.html"
        submitLink = f"/project/{projectID}/graph/task/{taskID}/"
        exitLink = f"/project/{projectID}/graph"
        deleteLink = f"/project/{projectID}/graph"
```